python/src/db/config.py

```diff
@@ -0,0 +1,15 @@
+import argparse
+
+
+def argument_parser():
+    parser = argparse.ArgumentParser(add_help=False)
```

Comment:
This implementation is really interesting, to make it more interesting why not implement the (Python-dotenv)[https://pypi.org/project/python-dotenv/]?

Seems a really good place to store default values without change it on code and of course you can configure the default if the dev forget to set the env vars.